website/cue/reference/releases/0.25.0.cue

```diff
@@ -9,15 +9,15 @@ releases: "0.25.0": {
 	known_issues: [
 		"""
 			Vector fails to load configurations using environment variables outside of string values
-			for configuration options. This will be fixed in 0.25.1.
+			for configuration options. Fixed in 0.25.1.
 			""",
 		"""
 			Vector fails to load multi-file configurations using the global `timezone` configuration
-			option. This will be fixed in 0.25.1.
+			option. Fixed in 0.25.1.
 			""",
 		"""
 			The `prometheus_remote_write` sink doesn't support specifying `auth.bearer` as it
-			should. This will be fixed in 0.25.1.
+			should. Fixed in 0.25.1.
```

Comment:
Perhaps the same on the line above this though I can't suggest since it wasn't edited here.